streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamAggregate.java

```diff
@@ -64,7 +64,8 @@ public void init(ProcessorContext context) {
 
         @Override
         public void process(K key, V value) {
-            if (key == null)
+        	    // If the key or value is null we don't need to proceed
+            if (key == null || value == null)
```

Comment:
Nit: can you clean up the code further and add `{ }` (we prefer to use `{}` for all blocks. Thanks.

Reply:
"can you clean up the code further " - please advise